lib/etl_cases.rb

```diff
@@ -0,0 +1,34 @@
+require 'generator/exercise_cases'
+
+class EtlCase < ExerciseCase
+  def workload
+    [
+      "old = #{format_hash(integerize_keys(input))}",
+      "    expected = #{format_hash(expected)}\n",
+      indent(4, assertion),
+    ].join("\n")
+  end
+
+  private
+
+  def indent(size, text)
+    text.lines.each_with_object('') { |line, obj| obj << ' ' * size + line }
+  end
```

Comment:
It has some magic indenting code, `indent_lines`. 

I've started converting `#workload` methods over to using it when I have a reason to rewrite a `#workload` method.